docs/src/howto-guides/010_add_self_signed_trusted.md

```diff
@@ -0,0 +1,27 @@
+# How to add self-signed certificate to trusted
+
+To manually add a self-signed certificate to be trusted by Linux system:
+
+Create a `/usr/local/share/ca-certificates/` directory if it does not exist on your computer:
+
+```shell
+mkdir /usr/local/share/ca-certificates/
+```
+
+Copy your root certificate (.crt file) to the created directory:
```

Comment:
Does the certificate file have to be in a specific format? PEM / DER / etc. ?

Reply:
They should be PEM crt files.

Reply:
Then let's say that. In my opinion, `.crt` doesn't imply PEM. If you want to do that with an extension, use `.pem` - but it's better to just state the format. Does it work with DER format too? If so, we should say that's possible. Otherwise customers think they have to go and work out how to convert it. What formats are valid?

Reply:
The provided method works for PEM certificates with explicit extension .crt as `update-ca-certificates` expects those to be like this. Do you want to include a manual method to add the certificates?

Reply:
Then it should say that they have to be in PEM format - or state what formats are allowed. Also call out more clearly that the extension must be `.crt`